store/index.js

```diff
@@ -10,25 +10,62 @@ const parseCookies = (cookie, key) => {
 }
 
 export const state = () => ({
-  auth: Cookie.get('auth'),
+  token: Cookie.get('token'),
+  id: Cookie.get('id'),
+  name: Cookie.get('name'),
+  email: Cookie.get('email'),
+  logo: Cookie.get('logo'),
 })
 
 export const mutations = {
-  setAuth(state, auth) {
-    Cookie.set('auth', auth)
-    state.auth = auth
+  setUserData(state, userData) {
+    if (!userData) {
+      removeCookie()
+      return
+    }
+    setCookie(state, userData)
   },
 }
 
+const removeCookie = () => {
+  Cookie.remove('token')
+  Cookie.remove('id')
+  Cookie.remove('name')
+  Cookie.remove('email')
+  Cookie.remove('logo')
+}
+
+const setCookie = (state, userData) => {
+  const { token, data: { id, name, email, logo } = {} } = userData || {}
+  Cookie.set('token', token)
+  Cookie.set('id', id)
+  Cookie.set('name', name)
```

Comment:
Store the user information as an object using a unique key on cookie.

Reply:
![image](https://user-images.githubusercontent.com/1880799/73508871-13018200-43bc-11ea-8581-3f9305bad5b2.png)
Tried to get the auth response to JSON.stringify and got those %20 :/
You know a solution to this? If not, let's get that merged and changed later on.